price-feeder/oracle/provider/provider.go

```diff
@@ -19,9 +19,15 @@ const (
 
 var ping = []byte("ping")
 
+// providerCandlePeriod represents the cutoff in minutes for old candles.
+const (
+	providerCandlePeriod = 10
```

Comment:
Maybe put it together with the others const and one idea is to set this as `time.Duration` like `providerCandlePeriod = 10 * time.Minute`

This way `PastUnixTime` could receive time.Duration instead of int64

Reply:
I like it :)

Reply:
bbe77fc32310f8bb9ea35bdf6777b849691c1edb 👍